IndicoV2.IntegrationTests/Reviews/ReviewsClientTests.cs

```diff
@@ -14,46 +17,59 @@ public class ReviewsClientTests
     {
         private IReviewsClient _reviewsClient;
         private DataHelper _dataHelper;
+        private ISubmissionResultAwaiter _submissionResultAwaiter;
+        private JobAwaiter _jobAwaiter;
 
         [SetUp]
         public void SetUp()
         {
-            var container = new IndicoTestContainerBuilder().Build();
+            var container = new IndicoTestContainerBuilder()
+                .ForAutoReviewWorkflow()
+                .Build();
             _dataHelper = container.Resolve<DataHelper>();
             _reviewsClient = container.Resolve<IReviewsClient>();
-        }
-
-        [Test]
-        public async Task RejectAsync_ShouldReturnJobId()
-        {
-            // Arrange
-            var submissionId = await GetSubmissionIdWithAutoReview();
-            
-            // Act
-            var jobId = await _reviewsClient.RejectAsync(submissionId, default);
-
-            // Assert
-            jobId.Should().NotBeNullOrEmpty();
-            Assert.Inconclusive("Requires JobsClient to verify job's result");
+            _submissionResultAwaiter = container.Resolve<ISubmissionResultAwaiter>();
+            _jobAwaiter = container.Resolve<JobAwaiter>();
         }
 
         [Test]
         public async Task SubmitReviewAsync_ShouldReturnJobId()
         {
             // Arrange
-            var submission = await GetSubmissionIdWithAutoReview();
+            var submission = await _dataHelper.Submissions().GetAnyAsync();
+            var result = await _submissionResultAwaiter.WaitReady(submission.Id);
+            var changes = (JObject)result["results"]["document"]["results"];
+
+            // as in: https://indicodatasolutions.github.io/indico-client-python/auto-review.html?highlight=submitreview
+            foreach (var (_, predictions) in changes)
+            {
+                foreach (var prediction in predictions.Value<JArray>("pre_review"))
+                {
+                    var label = prediction.Value<string>("label");
+                    if (prediction["confidence"].Value<double>(label) < 0.6d)
+                    {
+                        prediction["rejected"] = true;
+                    }
+                }
+            }
 
             // Act
-            var jobId = await _reviewsClient.SubmitReviewAsync(submission, JObject.Parse("{}"), default);
+            var jobId = await _reviewsClient.SubmitReviewAsync(submission.Id, changes);
+            try
+            {
+                var jobResult = await _jobAwaiter.WaitReadyAsync(jobId, TimeSpan.FromSeconds(1));
+                
+                // Assert
+                jobResult.Should().NotBeNullOrEmpty();
+            }
+            catch (JobNotSuccessfulException)
+            {
+                // TODO: fix the review
+                // Jobs's errorr: "Extraction predictions must be a list"
+                Assert.Fail("TODO: investigate what's wrong");
```

Comment:
I guess that we're leaving that for now, right?

Reply:
I'm waiting on clarification of how to make this work.